task-services/src/main/java/com/taat/taskservices/controllers/UserController.java

```diff
@@ -16,7 +18,16 @@ public class UserController {
   @Autowired
   private UserRepository userRepository;
 
-  @CrossOrigin(origins = { "http://localhost:3000", "https://onetaat-web.onrender.com" })
+  @GetMapping("/user")
+  public ResponseEntity<?> getUser(@AuthenticationPrincipal OAuth2User user) {
+    if (user == null) {
+      return new ResponseEntity<>("", HttpStatus.OK);
+    } else {
+      return ResponseEntity.ok().body(user.getAttributes());
+    }
+  }
+
+//  @CrossOrigin(origins = { "http://localhost:3000", "https://onetaat-web.onrender.com" })
```

Comment:
Being able to get rid of these annotations at the method- or class-level is great!